tests/kernels/test_cache.py

```diff
@@ -5,6 +5,7 @@
 import torch
 
 from vllm import _custom_ops as ops
+from vllm._C import cache_ops
```

Comment:
can you refactor `reshape_and_cache_flash` into `_custom_ops.py` to make the interface unified?